client/apps/edit/components/admin/test/verticals_tags.coffee

```diff
@@ -96,3 +96,65 @@ describe 'AdminVerticalsTags', ->
       @component.state.vertical.should.eql { name: 'Art', id: '1' }
       @component.props.onChange.args[0][0].should.eql 'vertical'
       @component.props.onChange.args[0][1].should.eql { name: 'Art', id: '1' }
+
+    it 'Unsets the vertical if already selected', ->
+      @component.setState vertical: { id: '1', name: 'Art'}
+      r.simulate.click r.find(@component, 'avant-garde-button')[0]
+      @component.props.onChange.args[0].should.eql [ 'vertical', null ]
+
+describe 'AdminTags', ->
+
+  beforeEach (done) ->
+    benv.setup =>
+      benv.expose
+        $: benv.require 'jquery'
+        _: benv.require 'underscore'
+        _s: benv.require 'underscore.string'
+      window.jQuery = $
+      @AdminTags = benv.require resolve __dirname, '../verticals_tags/index.coffee'
+      @AdminTags.__set__ 'sd', {}
+      @article = new Article
+      @article.attributes = fixtures().articles
+      @article.set 'tags', ['Artists']
+      @article.set 'tracking_tags', ['op-ed', 'profile']
```

Comment:
Backbone lets us assign attributes at initialization. I think it makes a little more sense to do something along the lines of:
```
@article = new Article _.extend fixtures().articles,
   tags: ['Artists']
   tracking_tags: ['op-ed', 'profile']
```